### PR TITLE
fix: add check helpers in Podman extension

### PR DESCRIPTION
**Pull request by jeffmaury**
### What does this PR do?

Add two new helper classes for Podman extension checks

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#6207 

### How to test this PR?

Run unit tests

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Review by benoitf — COMMENTED** [review]
hello, did you forget to commit the test file ? as the test checkbox is checked

**Comment by benoitf** [comment]
if you could update the PR's title, thx

**Comment by jeffmaury** [comment]
> hello, did you forget to commit the test file ? as the test checkbox is checked

Should be fixed now

**Comment by jeffmaury** [comment]
> if you could update the PR's title, thx

Done

File: extensions/podman/src/base-check.spec.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (benoitf) [inline extensions/podman/src/base-check.spec.ts]:
```suggestion
```

**Review by benoitf — APPROVED** [review]

**Review by axel7083 — APPROVED** [review]
LGTM

**Pull request merged**